app/services/populate_issues.rb

```diff
@@ -0,0 +1,30 @@
+class PopulateIssues
```

Comment:
Yeah, that's pretty much where I landed while live-streaming my thoughts above :)